src/constants/cookieNameKey.js

```diff
@@ -10,5 +10,5 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-export const EDGE_DOMAIN = "edge.adobedc.net";
-export const ID_THIRD_PARTY_DOMAIN = "adobedc.demdex.net";
+export const IDENTITY = "identity";
+export const CONSENT = "consent";
```

Comment:
This diff is confusing. `domains.js` got renamed to `domain.js` and the name of the constants changed.